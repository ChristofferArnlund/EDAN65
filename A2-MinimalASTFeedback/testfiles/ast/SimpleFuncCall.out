Program
  List
    Func
      IdDecl
        Name ID="a"
        Opt
      List
      StmtBlock
        List
          FuncCallStmt
            FuncCall
              IdUse
                Name ID="printf"
              List
