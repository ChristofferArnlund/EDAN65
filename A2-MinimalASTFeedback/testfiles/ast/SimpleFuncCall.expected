Program
  List
    Func
      Name ID="a"
      List
      List
        FuncCall
          Name ID="printf"
          List
